src/testFixtures/kotlin/uk/gov/justice/digital/hmpps/educationandworkplanapi/app/client/prisonapi/PrisonerInPrisonSummaryBuilder.kt

```diff
@@ -120,55 +121,55 @@ fun aValidPrisonPeriod(
   prisons: List<String> = listOf("BMI", "MDI", "BXI"),
   bookNumber: String = "1234A",
   bookingId: Long = 1,
-  releaseDate: OffsetDateTime? = null,
+  releaseDate: LocalDateTime? = null,
 ): PrisonPeriod =
   PrisonPeriod(
-    entryDate = entryDate,
+    entryDate = entryDate.asIsoDateTimeString()!!,
```

Comment:
The trade-off of reverting the swagger spec is that the generated model objects now contain Strings instead of OffsetDateTimes.